src/components/Accordion.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import './accordion.css';
+
+class Accordion extends Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            open: false
+        }
+
+        this.togglePanel = this.togglePanel.bind(this);
+        }
+        
+        togglePanel(e){
+        this.setState({open: !this.state.open})
```

Comment:
aici ii mai safe sa folosesti versiunea cu functie in this.setState

`this.setState(prevState => ({ open: !prevState.open }))`

In general daca folosesti in setState orice valoare precedenta/curenta din state ii mai bine sa folosesti versiunea asta de setState